modules/libcom/src/flex/flexdef.h

```diff
@@ -696,16 +696,16 @@ extern void dataend (void);
 extern void flexerror (char[]) NORETURN;
```

Comment:
This one already had `NORETURN`. So I used it for the other functions as well. Replace with `EPICS_NORETURN` to add the hints for Windows too?